dbt_subprojects/tokens/models/prices/degen/prices_degen_tokens.sql

```diff
@@ -0,0 +1,23 @@
+{% set blockchain = 'degen' %}
+
+{{ config(
+    schema = 'prices_' + blockchain,
+    alias = 'tokens',
+    materialized = 'table',
+    file_format = 'delta',
+    tags = ['static']
+    )
+}}
+
+SELECT
+    token_id
+    , '{{ blockchain }}' as blockchain
+    , symbol
+    , contract_address
+    , decimals
+FROM
+(
+    VALUES
+    ('degen-degen-base', 'DEGEN', 0x0000000000000000000000000000000000000000, 18)
```

Comment:
plz note, since we add here, we should *not* also add to `prices_native_tokens` or we will hit duplicates (it's not in there now, so should be fine)